packages/turbo-types/src/types/config.ts

```diff
@@ -192,11 +194,13 @@ export interface Pipeline {
   passThroughEnv?: null | Array<EnvWildcard>;
 
   /**
+   * Deprecated as of Turborepo 2.0.0. Consider using `inputs` instead.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#inputs
+   *
```

Comment:
```suggestion
```

Instead of linking to `inputs` within the TSDoc for `dotEnv` can we please remove this `Documentation: ` section?  That keeps it consistent and avoids any possible confusion about what the documentation is for (i.e. not this field).